it/controllers/pensions/paymentsIntoPensions/OneOffRASPaymentsAmountControllerISpec.scala

```diff
@@ -21,190 +21,103 @@ import builders.PensionsCYAModelBuilder._
-          formPostLinkCheck(oneOffReliefAtSourcePaymentsAmountUrl(taxYearEOY), formSelector)
-          welshToggleCheck(user.isWelsh)
-        }
-
-        "render Total one off payments into relief at source (RAS) pensions page prefilled when cya data" which {
-
-          val existingAmount: String = "999.88"
-          lazy val result: WSResponse = {
-            dropPensionsDB()
-            authoriseAgentOrIndividual(user.isAgent)
-            val pensionsViewModel = aPaymentsIntoPensionViewModel.copy(
-              oneOffRasPaymentPlusTaxReliefQuestion = Some(true), totalOneOffRasPaymentPlusTaxRelief = Some(BigDecimal(existingAmount)))
-            insertCyaData(pensionsUsersData(isPrior = false, aPensionsCYAModel.copy(paymentsIntoPension = pensionsViewModel)), aUserRequest)
-            urlGet(fullUrl(oneOffReliefAtSourcePaymentsAmountUrl(taxYearEOY)), user.isWelsh, follow = false,
-              headers = Seq(HeaderNames.COOKIE -> playSessionCookies(taxYearEOY, validTaxYearList)))
-          }
-
-          implicit def document: () => Document = () => Jsoup.parse(result.body)
-
-          "has an OK status" in {
-            result.status shouldBe OK
-          }
-
-          titleCheck(expectedTitle)
-          h1Check(expectedHeading)
-          captionCheck(expectedCaption(taxYearEOY), captionSelector)
-          textOnPageCheck(user.specificExpectedResults.get.expectedYouToldUs, paragraphSelector(1))
-          textOnPageCheck(expectedHowToWorkOut, paragraphSelector(2))
-          textOnPageCheck(expectedCalculationHeading, insetSpanText(1))
-          textOnPageCheck(expectedExampleCalculation, insetSpanText(2))
-          textOnPageCheck(hintText, hintTextSelector)
-          textOnPageCheck(poundPrefixText, poundPrefixSelector)
-          inputFieldValueCheck(amountInputName, inputSelector, existingAmount)
-          buttonCheck(buttonText, continueButtonSelector)
-          formPostLinkCheck(oneOffReliefAtSourcePaymentsAmountUrl(taxYearEOY), formSelector)
-          welshToggleCheck(user.isWelsh)
-
-        }
+      val existingAmount: String = "999.88"
+      lazy val result: WSResponse = {
+        dropPensionsDB()
+        authoriseAgentOrIndividual(isAgent = false)
+        val pensionsViewModel = aPaymentsIntoPensionViewModel.copy(
+          oneOffRasPaymentPlusTaxReliefQuestion = Some(true), totalOneOffRasPaymentPlusTaxRelief = Some(BigDecimal(existingAmount)))
+        insertCyaData(pensionsUsersData(aPensionsCYAModel.copy(paymentsIntoPension = pensionsViewModel)), aUserRequest)
+        urlGet(fullUrl(oneOffReliefAtSourcePaymentsAmountUrl(taxYearEOY)), follow = false,
+          headers = Seq(HeaderNames.COOKIE -> playSessionCookies(taxYearEOY, validTaxYearList)))
       }
+      "has an OK status" in {
+        result.status shouldBe OK
+      }
+      implicit def document: () => Document = () => Jsoup.parse(result.body)
+
+      import OneOffRASPaymentsAmountViewSpec._
```

Comment:
A general _suggestion_: rather than have the ISpec import from the ViewSpec, could we have both import from a common, helper class? Perhaps: `OneOffRASPaymentsAmountViewTestSupport`.

Reply:
Yeah, i named the object the same as the spec which probably isn't helpful

Reply:
Could we have a separate object, to be used by both?

Reply:
Confirmed that these have been moved into *TestSupport classes.